python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -408,18 +407,7 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         else:
             self.refObjLoader = None
 
-        dataRefDict['fgcmBuildStarsTableConfig'] = butlerQC.get(inputRefs.fgcmBuildStarsTableConfig)
-
-        fgcmBuildStarsConfig = butlerQC.get(inputRefs.fgcmBuildStarsTableConfig)
-        physicalFilterMap = fgcmBuildStarsConfig.physicalFilterMap
-
-        if self.config.doComposeWcsJacobian and not fgcmBuildStarsConfig.doApplyWcsJacobian:
-            raise RuntimeError("Cannot compose the WCS jacobian if it hasn't been applied "
-                               "in fgcmBuildStarsTask.")
-        if not self.config.doComposeWcsJacobian and fgcmBuildStarsConfig.doApplyWcsJacobian:
-            self.log.warn("Jacobian was applied in build-stars but doComposeWcsJacobian is not set.")
```

Comment:
We seem to be unable to read in the configs as regular inputs, and these checks seemed extraneous.  But my intention is to add these to the gen3 pipeline "contracts".  The pipeline itself isn't anywhere yet since it doesn't quite fit into the current framework; I'll ticket that separately.